src/contract.rs

```diff
@@ -676,11 +667,6 @@ fn cancel_ask(
     info: MessageInfo,
     id: String,
 ) -> Result<Response<ProvenanceMsg>, ContractError> {
-    // return error if id is empty
-    if id.is_empty() {
```

Comment:
was this moved to the request message validation?

Reply:
we have a `Uuid.parse` check

https://github.com/FigureTechnologies/ats-smart-contract/blob/main/src/msg.rs#L239-L243